config/initializers/commontator.rb

```diff
@@ -275,5 +274,5 @@
   # Default: lambda { |current_user, query|
   #                   current_user.class.where('username LIKE ?', "#{query}%") }
   config.user_mentions_proc = lambda { |current_user, query|
-    current_user.class.where('username LIKE ?', "#{query}%") }
+    current_user.class.where('name LIKE ?', "#{query}%") }
```

Comment:
We'll need something a bit more complicated than this because names contain spaces.